src/inspector/view/qml/MuseScore/Inspector/notation/notes/BeamSettings.qml

```diff
@@ -278,22 +271,24 @@ FocusableItem {
                             }
 
                             IncrementalPropertyControl {
-                                id: beamHeightLeftControl
+                                id: beamHeightRightControl
                                 anchors.left: lockButton.right
                                 anchors.leftMargin: 6
                                 anchors.right: parent.right
 
-                                icon: IconCode.BEAM_LEFT_Y_POSITION
+                                icon: IconCode.BEAM_HEIGHT_RIGHT
                                 iconMode: IncrementalPropertyControl.Right
-                                isIndeterminate: root.model ? root.model.beamVectorY.isUndefined : false
-                                currentValue: root.model ? root.model.beamVectorY.value : 0
+                                isIndeterminate: root.model ? root.model.beamHeightRight.isUndefined : false
+                                currentValue: root.model ? root.model.beamHeightRight.value : 0
 
-                                navigation.name: "BeamHightLeftControl"
+                                navigation.name: "BeamHeightRightControl"
                                 navigation.panel: root.navigationPanel
                                 navigation.row: lockButton.navigation.row + 1
-                                navigation.accessible.name: beamHeight.titleText + " " + qsTrc("inspector", "Left") + " " + currentValue
+                                navigation.accessible.name: beamHeight.titleText + " " + qsTrc("inspector", "Right") + " " + currentValue
 
-                                onValueEdited: function(newValue) { root.model.beamVectorY.value = newValue }
+                                onValueEdited: function(newValue) {
+                                    root.model.beamHeightRight.value = newValue
+                                }
```

Comment:
This appears to leave the "Force Horizontal" toggle as a large "flat button", which seems an odd control to use for a toggle.
I actually thought there was some logic in moving away from "Force horizontal" being a toggle, and rather simply adjusting the current stem heights so the beam became horizontal - the problem with the toggle in v3 is that you can turn it on, but then adjust the beam angle so it's no longer horizontal, but the "force horizontal" checkbox remains checked!  Is this addressed here?

Reply:
In MU3 that "force horizontal" remains check, but gets disabled is you modify the beam. That's OK IMHO. More probelmatic is that then "Userdefined position" also is disabled (and checked), yet the Position X/Y settings are used (and active)

Reply:
I agree that "Force horizontal" could as well be a checkbox, which will make its meaning clearer. We're still thinking about that a bit, but probably we'll change it. 

With this PR, using "Force horizontal" and "Userdefined position" are not mutually exclusive anymore, so both controls will be enabled all the time. They (should) work together nicely now, so that after choosing that you want the beam to be flat, you can still manually adjust its height.

Reply:
The new behaviour seems much better now to me, than in MU3.6.

The only remaining question for me is: if a beam has been manually moved, then 'Force horizontal' is clicked, the beam resets to its default position, and is then made horizontal. Perhaps the beam should, in this case, stay where it was moved, but be made horizontal? (I'm not really sure)

Reply:
I agree that would be better. Just resetting it was the easy option, but probably we can apply some smartness here. Will try it out.

Reply:
I replaced the Force horizontal button with a checkbox. 

About saving the modified beam height when toggling 'Force horizontal':
When you _enable_ it, then it is not difficult to keep the height and just make the beam flat. 
But when you _disable_ it, it seems difficult to go back to the old shape without fully resetting, because that is not stored anywhere. So the beam would stay flat, which may give the user the feeling that it didn't work. A solution could be to reset the position when the user _disables_ 'force horizontal', and keep it when the user _enables_ it. Is that okay?

Reply:
My first thought is that it's fine to leave the beam in its horizontal position when the user unchecks the box. The user is then a position where they can edit the endpoints independently, or they can easily manually reset the beam with Ctrl+R. 

I agree that it's a bit odd that seeing no change in the score when the option is turned off is a bit confusing, but having the beam also (potentially) drastically change position might also be confusing behaviour.

I can't say this is a particularly strong preference though. Maybe @Tantacrul or others have an opinion.

Reply:
Okay, so now I implemented that: when disabling Force horizontal on a custom-positioned beam, it will stay flat but allow customizations again.